### DISCUSSION TITLE
PLEASE update this =(

### DISCUSSION THREAD

**Discussion by Eddieeeeeeeeeeee** [m4dEngi/RemotePlayWhatever#168] [General]
Sir,
I have never used your code before but I very much need this to play games with a poor soul in Iran, who is running the shittiest laptop and has not many options of distracting herself from her shitty life...
Not sure if anyone wants to read the back story but let me assure you that I - and her - would be very, very thankful for your work.

Greetings, sir!

**Comment by ariser123** [comment]
Using this to play with my brothers I hope they update it soon

**Comment by Eddieeeeeeeeeeee** [comment]
It seems that prayers work after all.
Thank you so much, sir! =))))))))))))))))))))))))
